testsuite/objects/__init__.py

```diff
@@ -1,11 +1,50 @@
 """Module containing base classes for common objects"""
 import abc
-from dataclasses import dataclass
+from dataclasses import dataclass, is_dataclass, fields
+from copy import deepcopy
 from functools import cached_property
-from typing import Literal, List
+from typing import Literal, List, Dict, Union
```

Comment:
You can use generic types for `list` and `dict` instead (https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections).

Reply:
Removed types `List, Dict `and replaced by `list, dict`